test/sanity/validate-modules/validate-modules

```diff
@@ -851,6 +853,30 @@ class ModuleValidator(Validator):
 
         return False
 
+    def _check_for_subprocess(self):
+        for child in self.ast.body:
+            if isinstance(child, ast.Import):
+                if child.names[0].name == 'subprocess':
+                    for line_no, line in enumerate(self.text.splitlines()):
+                        sp_match = SUBPROCESS_REGEX.search(line)
+                        if sp_match:
+                            self.errors.append((
+                                209,
+                                ('subprocess.Popen call found. Should be module.run_command - %d:%d' %
+                                 (line_no + 1, sp_match.span()[0] + 1))
+                            ))
+
+    def _check_for_os_call(self):
+        if 'os.call' in self.text:
+            for line_no, line in enumerate(self.text.splitlines()):
```

Comment:
Just to add to my previous comment, this is actually the same thing as `if child.names[0].name == 'subprocess':`; they are just done in 2 different ways. Mainly because I couldn't find a a way to leverage `ast` to find `os.call()` calls. Again, more than happy to suggestions here :)